tee_supplicant/src/tee_supplicant.c

```diff
@@ -108,7 +208,7 @@ static int load_ta(uint32_t num_params, struct tee_param *params)
 		return TEEC_ERROR_ITEM_NOT_FOUND;
 	}
 
-	MEMREF_SIZE(params + 1) = size;
+	params[1].b = size;
```

Comment:
why drop this macro?

Reply:
Because it is unsafe to make MACRO(x) = y. I should be refactored at least to be SET_MEMREF_SIZE(x, y)

Reply:
but `MEMREF_SIZE` & `MEMREF_SHM_ID` & `MEMREF_SHM_OFFS` macros are all already used this way in `libteec/src/tee_client_api.c` (and same for reference Linux libteec)

Reply:
Macros usage restored